shanoir-ng-datasets/src/main/java/org/shanoir/ng/importer/service/ImporterService.java

```diff
@@ -120,18 +120,23 @@ public void createAllDatasetAcquisition(ImportJob importJob, Long userId) throws
 				for (Patient patient : importJob.getPatients()) {
 					for (Study study : patient.getStudies()) {
 						float progress = 0.5f;
+
+						int nbSeries = study.getSeries().size();
+						int cpt = 1;
+
 						for (Serie serie : study.getSeries() ) {
 							if (serie.getSelected() != null && serie.getSelected()) {
+								event.setMessage("Creating datasets for serie [" + serie.getProtocolName() + "] (" + cpt + "/" + nbSeries + ") of examination [" + importJob.getExaminationId() + "]...");
```

Comment:
Hi Youenn, it might be confusing for the user, when he selected 5 series for import from 10 series available to show him 1/10 series as part of the ongoing import. Either check before the number of selected series or just write the name of the serie currently under import. With kind regards, Michael